spacemesh/v2/reward.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package spacemesh.v2;
+
+message Reward {
+  oneof versioned {
+    RewardV1 v1 = 1;
+  }
+}
+
+message RewardV1 {
+  uint32 layer = 1; // layer award was paid in
+  uint64 total = 2; // total reward paid (sum of tx fee and layer reward)
+  uint64 layer_reward = 3; // tx_fee = total - layer_reward
+  uint32 layer_computed = 4; // layer number of the layer when reward was computed
+  string coinbase = 5; // account awarded this reward
+  bytes smesher = 6;  // id of smesher who earned this reward
```

Comment:
bytes or string format?

Reply:
As we use string for address, I will change that to string as well